.github/workflows/build.yml

```diff
@@ -194,13 +203,19 @@ jobs:
     - name: Build some applications (arm64)
       if: ${{ matrix.arch == 'arm64' }}
       run: cmake --build build --target calc magnify mstsc notepad osk regedit taskmgr winmine wordpad base/applications/screensavers/all -- -k0
+    - name: Build control panel applets
```

Comment:
```suggestion
    - name: Build control panel applets (arm64)
```

Reply:
Right. Done.

--

Shall I do some merges?
- arm and arm64 dll/cpl/all together
- or all arm in 1 task (or at least all dll/* together) and all arm64 in 1 task,
with separate or unique cmake call

Reply:
Let's keep it this way for now.

>